nyaa/templates/layout.html

```diff
@@ -107,6 +107,15 @@
 						<li class="dropdown">
 							<a href="#" class="dropdown-toggle visible-lg visible-sm visible-xs" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
 								<i class="fa fa-user fa-fw"></i>
+								{% set notif_number = g.user.unread_notifications_number %}
+								{% if notif_number > 0 %}
```

Comment:
The docs state that if the badge is not visible when empty, so there's no real need to check beforehand.
Docs: http://getbootstrap.com/components/#badges

(sorry, removed by mistake)